diff -up lxqt-policykit-0.16.0/CMakeLists.txt.1~ lxqt-policykit-0.16.0/CMakeLists.txt
--- lxqt-policykit-0.16.0/CMakeLists.txt.1~	2020-11-06 03:39:56.206899767 +0100
+++ lxqt-policykit-0.16.0/CMakeLists.txt	2020-11-06 03:41:52.808086988 +0100
@@ -19,7 +19,7 @@ find_package(PkgConfig)
 pkg_check_modules(POLKIT_AGENT REQUIRED polkit-agent-1)
 message(STATUS "polkit agent: ${POLKIT_AGENT_INCLUDE_DIRS} ${POLKIT_AGENT_LIBRARIES}")
 
-set(POLKIT_AGENT_BINARY_DIR "${CMAKE_INSTALL_BINDIR}" CACHE FILEPATH "Directory for install polkit agent")
+set(POLKIT_AGENT_BINARY_DIR "${CMAKE_INSTALL_FULL_LIBEXECDIR}" CACHE FILEPATH "Directory for install polkit agent")
 
 # Minumum Versions
 set(LXQT_MINIMUM_VERSION "0.16.0")
