--- lxqt-policykit-20140803/CMakeLists.txt.omv~	2014-08-03 20:22:07.805118395 +0200
+++ lxqt-policykit-20140803/CMakeLists.txt	2014-08-03 20:24:30.399439912 +0200
@@ -10,6 +10,10 @@ option(USE_QT5 "Build with Qt5." $ENV{US
 # find Qt binding of polkit
 if(USE_QT5)
     find_package(PolkitQt5-1) # Qt5 version
+    set(POLKITQT-1_VERSION_STRING "${POLKITQT5-1_VERSION_STRING}")
+    set(POLKITQT-1_INCLUDE_DIR "${POLKITQT5-1_INCLUDE_DIR}")
+    set(POLKITQT-1_AGENT_LIBRARY "${POLKITQT5-1_AGENT_LIBRARY}")
+    set(POLKITQT-1_CORE_LIBRARY "${POLKITQT5-1_CORE_LIBRARY}")
 else()
     find_package(PolkitQt-1) # Qt4 version
 endif()
