diff -up lxqt-policykit-0.17.0/autostart/lxqt-policykit-agent.desktop.in.2~ lxqt-policykit-0.17.0/autostart/lxqt-policykit-agent.desktop.in
--- lxqt-policykit-0.17.0/autostart/lxqt-policykit-agent.desktop.in.2~	2021-04-15 19:26:39.059877729 +0200
+++ lxqt-policykit-0.17.0/autostart/lxqt-policykit-agent.desktop.in	2021-04-15 19:53:00.961468968 +0200
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Type=Application
-TryExec=lxqt-policykit-agent
-Exec=lxqt-policykit-agent
+TryExec=/usr/libexec/lxqt-policykit-agent
+Exec=/usr/libexec/lxqt-policykit-agent
 OnlyShowIn=LXQt;
 X-LXQt-Module=true
 
