diff -up lxqt-policykit-2.1.0/CMakeLists.txt.1~ lxqt-policykit-2.1.0/CMakeLists.txt
--- lxqt-policykit-2.1.0/CMakeLists.txt.1~	2024-11-05 17:39:52.461213496 +0100
+++ lxqt-policykit-2.1.0/CMakeLists.txt	2024-11-05 17:44:37.014521156 +0100
@@ -19,7 +19,7 @@ find_package(PkgConfig)
 pkg_check_modules(POLKIT_AGENT REQUIRED polkit-agent-1)
 message(STATUS "polkit agent: ${POLKIT_AGENT_INCLUDE_DIRS} ${POLKIT_AGENT_LIBRARIES}")
 
-set(POLKIT_AGENT_BINARY_DIR "${CMAKE_INSTALL_BINDIR}" CACHE FILEPATH "Directory for install polkit agent")
+set(POLKIT_AGENT_BINARY_DIR "${CMAKE_INSTALL_FULL_LIBEXECDIR}" CACHE FILEPATH "Directory for install polkit agent")
 
 # Minumum Versions
 set(LXQT_MINIMUM_VERSION "2.1.0")
