diff -up lxqt-policykit-0.17.0/CMakeLists.txt.1~ lxqt-policykit-0.17.0/CMakeLists.txt
--- lxqt-policykit-0.17.0/CMakeLists.txt.1~	2021-04-15 19:19:56.279283339 +0200
+++ lxqt-policykit-0.17.0/CMakeLists.txt	2021-04-15 19:26:11.963397646 +0200
@@ -19,7 +19,7 @@ find_package(PkgConfig)
 pkg_check_modules(POLKIT_AGENT REQUIRED polkit-agent-1)
 message(STATUS "polkit agent: ${POLKIT_AGENT_INCLUDE_DIRS} ${POLKIT_AGENT_LIBRARIES}")
 
-set(POLKIT_AGENT_BINARY_DIR "${CMAKE_INSTALL_BINDIR}" CACHE FILEPATH "Directory for install polkit agent")
+set(POLKIT_AGENT_BINARY_DIR "${CMAKE_INSTALL_FULL_LIBEXECDIR}" CACHE FILEPATH "Directory for install polkit agent")
 
 # Minumum Versions
 set(LXQT_MINIMUM_VERSION "0.17.0")
