diff -up lxqt-policykit-1.3.0/CMakeLists.txt.1~ lxqt-policykit-1.3.0/CMakeLists.txt
--- lxqt-policykit-1.3.0/CMakeLists.txt.1~	2023-04-16 23:47:34.918309595 +0200
+++ lxqt-policykit-1.3.0/CMakeLists.txt	2023-04-16 23:49:10.017197828 +0200
@@ -19,7 +19,7 @@ find_package(PkgConfig)
 pkg_check_modules(POLKIT_AGENT REQUIRED polkit-agent-1)
 message(STATUS "polkit agent: ${POLKIT_AGENT_INCLUDE_DIRS} ${POLKIT_AGENT_LIBRARIES}")
 
-set(POLKIT_AGENT_BINARY_DIR "${CMAKE_INSTALL_BINDIR}" CACHE FILEPATH "Directory for install polkit agent")
+set(POLKIT_AGENT_BINARY_DIR "${CMAKE_INSTALL_FULL_LIBEXECDIR}" CACHE FILEPATH "Directory for install polkit agent")
 
 # Minumum Versions
 set(LXQT_MINIMUM_VERSION "1.3.0")
